### ISSUE TITLE
Fix release proces - meta-dts has de-synced information on changelog and release notes tab

### ISSUE THREAD

**Issue by artur-rs** [zarhus/zarhus-issues#29]
* https://github.com/Dasharo/meta-dts/releases
* https://github.com/Dasharo/meta-dts/blob/main/CHANGELOG.md

1. Fix relese-notes information on the above resources
2. Add required steps to the standard release process to avoid such problems in the future

**Comment by m-iwanicki** [comment]
@artur-rs Is this correct repository for this issue?

**Comment by artur-rs** [comment]
@m-iwanicki not exactly, probably wrong tab during issue creation. I'll move it if there is such feature

**edit:** related issues /PR:
- https://github.com/Dasharo/dasharo-issues/issues/1185
- https://github.com/Dasharo/meta-dts/pull/225
- https://github.com/Dasharo/meta-dts/pull/152

I suppose we can close this one as a duplicate/already addressed.
